tests/integration/pass/strings.ncl

```diff
@@ -12,17 +12,17 @@ let {check, ..} = import "testlib.ncl" in
     == "Hello, world! Welcome in the world-universe",
 
   # regression test for issue #361 (https://github.com/tweag/nickel/issues/361)
-  m%""%{"foo"}""%m == "\"foo\"",
-  m%"""%m == "\"",
-  m%""%"%"%"%m == "\"%\"%\"%",
+  m%""%{"foo"}""% == "\"foo\"",
+  m%"""% == "\"",
+  m%%""%"%"%"%% == "\"%\"%\"%",
```

Comment:
Yes, the extra `%` is deliberate. The input `m%""%"%"%"%` is no longer a valid program, as it parses as `m%""% | "%" | %"%`, since the first `"%` we encounter closes the string. 

You're right about this changing the nature of the test, though my understanding of the original test was that it was ensuring that `"%` not followed by `m` _doesn't_ close the string, which is no longer true. I agree that it also validated the behaviour you describe though. 

I'll replace it with something like the following, which tests the (lack of) interpolation:

```
m%"some text %not-interpolated"% == "some text %not-interpolated",
m%%"%%also-not-interpolated"% == "%%also-not-interpolated",
```

Reply:
Ah, of course, I didn't think about the presence of `"`.